Add UI information about AAQ operator (#111679)
openshift/openshift-docs: modules/virt-enabling-aaq-operator.adoc

// Module included in the following assemblies:
//
// * virt/managing_vms/advanced_vm_management/virt-working-with-resource-quotas-for-vms.adoc

:_mod-docs-content-type: PROCEDURE
[id="virt-enabling-aaq-operator_{context}"]
= Enabling the Application Aware Quota Operator

[role="_abstract"]
Enable the Application Aware Quota (AAQ) operator to manage resource quotas to extend native resource management capabilities. Enabling AAQ from the web console eliminates the need to manually edit the HyperConverged custom resource by using the CLI.

.Prerequisites

> * You have administrator privileges in the OpenShift cluster.

Reviewer: Optional, for consistency
```suggestion
* You have cluster administrator privileges.
```

* The OpenShift Virtualization operator is installed and running.

.Procedure

> . In the OpenShift web console, click *Virtualization* -> *Settings*.

Reviewer: ```suggestion
. In the {product-title} web console, click *Virtualization* -> *Settings*.
```

// Module included in the following assemblies:
//
// * virt/managing_vms/advanced_vm_management/virt-working-with-resource-quotas-for-vms.adoc

:_mod-docs-content-type: PROCEDURE
[id="virt-enabling-aaq-operator_{context}"]
= Enabling the Application Aware Quota Operator

[role="_abstract"]
Enable the Application Aware Quota (AAQ) operator to manage resource quotas to extend native resource management capabilities. Enabling AAQ from the web console eliminates the need to manually edit the HyperConverged custom resource by using the CLI.

.Prerequisites

* You have administrator privileges in the OpenShift cluster.
* The OpenShift Virtualization operator is installed and running.

.Procedure

> . In the OpenShift web console, click *Virtualization* -> *Overview*.

Reviewer 2: Virtualization navigation has changed since console 4.22, the correct path is now `*Virtualization* -> *Settings*` (and then there is no Settings tab)

Author: ```suggestion
. In the OpenShift web console, click *Virtualization* -> *Settings*.
```

> . Click the *Settings* tab.

Author: ```suggestion
```
